README.md

```diff
@@ -137,6 +137,21 @@ Onfido.init({
   containerId: 'onfido-mount',
   onComplete: function(data) {
     console.log("everything is complete")
+    // `data` will be an object that looks something like this:
+    // ```
+    // {
+    //   "document_front": {
+    //     "id": "5c7b8461-0e31-4161-9b21-34b1d35dde61",
+    //     "type": "passport",
+    //     "side": "front"
```

Comment:
Do we need to return the side given that the key is `document_front`?

Reply:
As discussed offline, I agree we should return the side and two objects for each side captured.